client/src/components/atoms/AddPeopleButton/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { AddPeople } from "~/shared/icons/AddPeople";
+
+type AddPeopleButtonType = {
+  onClick: () => void;
+  isSolid?: boolean;
+  isHeader?: boolean
+}
+
+const AddPeopleButton = ({ onClick, isSolid = true, isHeader = true }: AddPeopleButtonType) => {
+  return (
```

Comment:
1. Change `AddPeopleButtonType` to `Props` since the purpose of this type in for the component only and is not being re-used in other pages
2. You can change this to `const AddPeopleButton: React.FC<Props> = ({ onClick, isSolid = true, isHeader = true }): => {}`
Change this to other components as well